src/System.Diagnostics.DiagnosticSource/src/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      net45-Windows_NT;
```

Comment:
The feature implemented here will be used in ASP.NET Core, which also targets .NET 4.5.1
Some APIs (CallContext, which is crucial for the feature) are **not** available on netstandard1.1 and available on .NET 4.5.